routes/api/articles.js

```diff
@@ -297,4 +303,83 @@ router.get(
   commentController.findCommentThreadController
 );
 
+/**
+ * @swagger
+ *
+ * /:articleId/:reaction:
+ *    put:
+ *     description: React to a particular article
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: articleId
+ *         description: The id of an existing article
+ *         in:  request
+ *       - name: reaction
+ *         description: A like or a dislike
+ *         in:  query
+ *     responses:
+ *       - 200:
+ *         message: article has been liked/disliked
+ *       - 500:
+ *         message: Internal server error
+ *       - 404:
+ *         message: article not found
+ *
+ *
+ */
+router.put('/reaction/:articleId/', jwtValidator, verifyArticleId, validateReaction, likeOrDislike);
+
+/**
+ * @swagger
+ *
+ * /:articleId/:reaction:
+ *     get:
+ *     description: Get number of likes or dislikes for an article
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: articleId
+ *         description: The id of an existing article
+ *         in:  request
+ *       - name: reaction
+ *         description: like or a dislike
+ *         in:  query
+ *     responses:
+ *       - 200:
+ *         message: Total number of likes/dislikes for this article
+ *         data: {count:[number of likes or dislikes]}
+ *       - 500:
+ *         message: Internal server error
+ *       - 404:
+ *         message: article not found
+ *
+ *
+ */
+router.get('/reaction/:articleId/', verifyArticleId, validateReaction, allReactions);
+
+/**
+ * @swagger
+ *
+ * /:articleId/:reaction:
+ *     delete:
+ *     description: Delete an existing reaction to an article
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: reactionId
+ *         description: The id of an existing reaction
+ *         in:  request
+ *     responses:
+ *       - 200:
+ *         message: Reaction removed successfully
+ *       - 500:
+ *         message: Internal server error
+ *       - 404:
+ *         message: Invalid reaction Id
+ *
+ *
+ */
+router.delete('/reaction/:reactionId', jwtValidator, cancelReaction);
```

Comment:
The endpoint for this would be `api/v1/comments/some_reactionn_id` this seems to me like you are deleting a comment. Having the endpoint this way `api/v1/comments/reaction/some_reactionn_id`, clearly describes what is going on here.

Reply:
A server error is thrown when a reaction id is not valid. Please validate reaction id. Please do the same for every endpoint where ID is required.

Reply:
I have made all necessary changes. Thanks

Reply:
`api/v1/comments/reaction/some_reactionn_id` for comment reaction
`api/v1/articles/reaction/some_reactionn_id` for article reaction